files/aiida-core-constraints.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
```

Comment:
Fine to keep it here for the moment - eventually one could think about adding this e.g. in https://github.com/aiidateam/aiida-core/tree/develop/utils